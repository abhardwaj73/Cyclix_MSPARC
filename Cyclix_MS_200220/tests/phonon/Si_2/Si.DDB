***************************************************************************
                            Phononic properties                            
***************************************************************************

Dynamical matrix before symmetrization (not shifted) (Ha/Bohr^2) for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.2427535498 +      -0.0000000000i	      0.0043115603 +       0.0000000000i	      0.0223176823 +      -0.0000000000i	      0.2048708755 +      -0.0000000000i	      0.0045489993 +       0.0000000000i	      0.0539533339 +       0.0000000000i	     -0.0853693803 +       0.0000000000i	     -0.0215866156 +      -0.0000000000i	     -0.0031220050 +      -0.0000000000i	      0.1282724216 +       0.0000000000i	      0.0211250906 +      -0.0000000000i	     -0.0731741197 +       0.0000000000i	
      0.0043115603 +       0.0000000000i	     -0.7576241965 +       0.0000000000i	     -0.1548325480 +      -0.0000000000i	     -0.0264475588 +      -0.0000000000i	      0.8147110528 +       0.0000000000i	      0.1563064271 +       0.0000000000i	      0.1752189878 +       0.0000000000i	     -0.7437270704 +       0.0000000000i	      0.1560653478 +      -0.0000000000i	     -0.1102034033 +       0.0000000000i	      0.6962791402 +      -0.0000000000i	     -0.1550623924 +       0.0000000000i	
      0.0223176823 +      -0.0000000000i	     -0.1548325480 +      -0.0000000000i	     -0.1700446198 +      -0.0000000000i	     -0.0534091946 +       0.0000000000i	      0.0230871580 +       0.0000000000i	     -0.1073284293 +       0.0000000000i	      0.0009569852 +       0.0000000000i	     -0.0232854295 +       0.0000000000i	      0.1241579828 +       0.0000000000i	      0.0755953089 +       0.0000000000i	     -0.0077435146 +      -0.0000000000i	      0.1531794104 +      -0.0000000000i	
      0.2048708755 +      -0.0000000000i	     -0.0264475588 +       0.0000000000i	     -0.0534091946 +      -0.0000000000i	     -0.2428165628 +       0.0000000000i	      0.0045197310 +      -0.0000000000i	     -0.0222694547 +      -0.0000000000i	      0.1281800830 +      -0.0000000000i	      0.0208910223 +       0.0000000000i	      0.0731904792 +      -0.0000000000i	     -0.0852777053 +      -0.0000000000i	     -0.0213636919 +       0.0000000000i	      0.0030343837 +      -0.0000000000i	
      0.0045489993 +       0.0000000000i	      0.8147110528 +       0.0000000000i	      0.0230871580 +       0.0000000000i	      0.0045197310 +      -0.0000000000i	     -0.8066730169 +      -0.0000000000i	      0.1109897268 +      -0.0000000000i	     -0.1284996318 +       0.0000000000i	      0.7426514592 +       0.0000000000i	      0.1111528815 +      -0.0000000000i	      0.1540344994 +       0.0000000000i	     -0.6839065257 +      -0.0000000000i	     -0.1121904730 +       0.0000000000i	
      0.0539533339 +       0.0000000000i	      0.1563064271 +       0.0000000000i	     -0.1073284293 +       0.0000000000i	     -0.0222694547 +      -0.0000000000i	      0.1109897268 +      -0.0000000000i	     -0.1700919494 +      -0.0000000000i	     -0.0757288953 +      -0.0000000000i	      0.0083808540 +      -0.0000000000i	      0.1532065313 +       0.0000000000i	     -0.0008072080 +      -0.0000000000i	      0.0226254784 +       0.0000000000i	      0.1242051769 +      -0.0000000000i	
     -0.0853693803 +       0.0000000000i	      0.1752189878 +       0.0000000000i	      0.0009569852 +       0.0000000000i	      0.1281800830 +      -0.0000000000i	     -0.1284996318 +      -0.0000000000i	     -0.0757288953 +      -0.0000000000i	     -0.2615152948 +       0.0000000000i	      0.1129986986 +      -0.0000000000i	      0.0435192933 +       0.0000000000i	      0.2282836472 +       0.0000000000i	     -0.0997840458 +      -0.0000000000i	      0.0310455069 +      -0.0000000000i	
     -0.0215866156 +      -0.0000000000i	     -0.7437270704 +      -0.0000000000i	     -0.0232854295 +      -0.0000000000i	      0.0208910223 +       0.0000000000i	      0.7426514592 +       0.0000000000i	      0.0083808540 +       0.0000000000i	      0.1129986986 +      -0.0000000000i	     -0.7881398706 +      -0.0000000000i	     -0.1046166308 +       0.0000000000i	     -0.1217333238 +       0.0000000000i	      0.7282559050 +      -0.0000000000i	      0.1205887091 +      -0.0000000000i	
     -0.0031220050 +      -0.0000000000i	      0.1560653478 +      -0.0000000000i	      0.1241579828 +       0.0000000000i	      0.0731904792 +       0.0000000000i	      0.1111528815 +      -0.0000000000i	      0.1532065313 +       0.0000000000i	      0.0435192933 +       0.0000000000i	     -0.1046166308 +       0.0000000000i	     -0.1708630980 +      -0.0000000000i	     -0.0566750367 +       0.0000000000i	      0.0105830566 +      -0.0000000000i	     -0.1064056676 +       0.0000000000i	
      0.1282724216 +       0.0000000000i	     -0.1102034033 +       0.0000000000i	      0.0755953089 +       0.0000000000i	     -0.0852777053 +      -0.0000000000i	      0.1540344994 +      -0.0000000000i	     -0.0008072080 +      -0.0000000000i	      0.2282836472 +       0.0000000000i	     -0.1217333238 +       0.0000000000i	     -0.0566750367 +      -0.0000000000i	     -0.2578902969 +       0.0000000000i	      0.1037712077 +      -0.0000000000i	     -0.0521922876 +      -0.0000000000i	
      0.0211250906 +      -0.0000000000i	      0.6962791402 +      -0.0000000000i	     -0.0077435146 +      -0.0000000000i	     -0.0213636919 +       0.0000000000i	     -0.6839065257 +       0.0000000000i	      0.0226254784 +       0.0000000000i	     -0.0997840458 +      -0.0000000000i	      0.7282559050 +      -0.0000000000i	      0.0105830566 +       0.0000000000i	      0.1037712077 +      -0.0000000000i	     -0.7427786956 +       0.0000000000i	      0.1477414698 +       0.0000000000i	
     -0.0731741197 +       0.0000000000i	     -0.1550623924 +       0.0000000000i	      0.1531794104 +      -0.0000000000i	      0.0030343837 +      -0.0000000000i	     -0.1121904730 +       0.0000000000i	      0.1242051769 +      -0.0000000000i	      0.0310455069 +      -0.0000000000i	      0.1205887091 +      -0.0000000000i	     -0.1064056676 +       0.0000000000i	     -0.0521922876 +      -0.0000000000i	      0.1477414698 +       0.0000000000i	     -0.1709633849 +      -0.0000000000i	

Dynamical matrix after symmetrization (not shifted) (Ha/Bohr^2) for q = [      0.0000000000       0.0000000000       0.0000000000]:
     -0.2427535498 +       0.0000000000i	      0.0043115603 +      -0.0000000000i	      0.0223176823 +       0.0000000000i	      0.2048708755 +       0.0000000000i	      0.0045489993 +      -0.0000000000i	      0.0539533339 +      -0.0000000000i	     -0.0853693803 +      -0.0000000000i	     -0.0215866156 +       0.0000000000i	     -0.0031220050 +       0.0000000000i	      0.1282724216 +      -0.0000000000i	      0.0211250906 +       0.0000000000i	     -0.0731741197 +      -0.0000000000i	
      0.0043115603 +       0.0000000000i	     -0.7576241965 +       0.0000000000i	     -0.1548325480 +       0.0000000000i	     -0.0264475588 +      -0.0000000000i	      0.8147110528 +      -0.0000000000i	      0.1563064271 +      -0.0000000000i	      0.1752189878 +       0.0000000000i	     -0.7437270704 +       0.0000000000i	      0.1560653478 +       0.0000000000i	     -0.1102034033 +       0.0000000000i	      0.6962791402 +      -0.0000000000i	     -0.1550623924 +      -0.0000000000i	
      0.0223176823 +      -0.0000000000i	     -0.1548325480 +      -0.0000000000i	     -0.1700446198 +       0.0000000000i	     -0.0534091946 +       0.0000000000i	      0.0230871580 +       0.0000000000i	     -0.1073284293 +       0.0000000000i	      0.0009569852 +       0.0000000000i	     -0.0232854295 +       0.0000000000i	      0.1241579828 +       0.0000000000i	      0.0755953089 +       0.0000000000i	     -0.0077435146 +      -0.0000000000i	      0.1531794104 +      -0.0000000000i	
      0.2048708755 +      -0.0000000000i	     -0.0264475588 +       0.0000000000i	     -0.0534091946 +      -0.0000000000i	     -0.2428165628 +       0.0000000000i	      0.0045197310 +       0.0000000000i	     -0.0222694547 +       0.0000000000i	      0.1281800830 +      -0.0000000000i	      0.0208910223 +      -0.0000000000i	      0.0731904792 +      -0.0000000000i	     -0.0852777053 +       0.0000000000i	     -0.0213636919 +      -0.0000000000i	      0.0030343837 +       0.0000000000i	
      0.0045489993 +       0.0000000000i	      0.8147110528 +       0.0000000000i	      0.0230871580 +      -0.0000000000i	      0.0045197310 +      -0.0000000000i	     -0.8066730169 +       0.0000000000i	      0.1109897268 +       0.0000000000i	     -0.1284996318 +       0.0000000000i	      0.7426514592 +       0.0000000000i	      0.1111528815 +      -0.0000000000i	      0.1540344994 +       0.0000000000i	     -0.6839065257 +      -0.0000000000i	     -0.1121904730 +       0.0000000000i	
      0.0539533339 +       0.0000000000i	      0.1563064271 +       0.0000000000i	     -0.1073284293 +      -0.0000000000i	     -0.0222694547 +      -0.0000000000i	      0.1109897268 +      -0.0000000000i	     -0.1700919494 +       0.0000000000i	     -0.0757288953 +      -0.0000000000i	      0.0083808540 +      -0.0000000000i	      0.1532065313 +      -0.0000000000i	     -0.0008072080 +      -0.0000000000i	      0.0226254784 +       0.0000000000i	      0.1242051769 +       0.0000000000i	
     -0.0853693803 +       0.0000000000i	      0.1752189878 +      -0.0000000000i	      0.0009569852 +      -0.0000000000i	      0.1281800830 +       0.0000000000i	     -0.1284996318 +      -0.0000000000i	     -0.0757288953 +       0.0000000000i	     -0.2615152948 +       0.0000000000i	      0.1129986986 +       0.0000000000i	      0.0435192933 +      -0.0000000000i	      0.2282836472 +      -0.0000000000i	     -0.0997840458 +       0.0000000000i	      0.0310455069 +       0.0000000000i	
     -0.0215866156 +      -0.0000000000i	     -0.7437270704 +      -0.0000000000i	     -0.0232854295 +      -0.0000000000i	      0.0208910223 +       0.0000000000i	      0.7426514592 +      -0.0000000000i	      0.0083808540 +       0.0000000000i	      0.1129986986 +      -0.0000000000i	     -0.7881398706 +       0.0000000000i	     -0.1046166308 +      -0.0000000000i	     -0.1217333238 +      -0.0000000000i	      0.7282559050 +       0.0000000000i	      0.1205887091 +       0.0000000000i	
     -0.0031220050 +      -0.0000000000i	      0.1560653478 +      -0.0000000000i	      0.1241579828 +      -0.0000000000i	      0.0731904792 +       0.0000000000i	      0.1111528815 +       0.0000000000i	      0.1532065313 +       0.0000000000i	      0.0435192933 +       0.0000000000i	     -0.1046166308 +       0.0000000000i	     -0.1708630980 +       0.0000000000i	     -0.0566750367 +       0.0000000000i	      0.0105830566 +      -0.0000000000i	     -0.1064056676 +      -0.0000000000i	
      0.1282724216 +       0.0000000000i	     -0.1102034033 +      -0.0000000000i	      0.0755953089 +      -0.0000000000i	     -0.0852777053 +      -0.0000000000i	      0.1540344994 +      -0.0000000000i	     -0.0008072080 +       0.0000000000i	      0.2282836472 +       0.0000000000i	     -0.1217333238 +       0.0000000000i	     -0.0566750367 +      -0.0000000000i	     -0.2578902969 +       0.0000000000i	      0.1037712077 +       0.0000000000i	     -0.0521922876 +       0.0000000000i	
      0.0211250906 +      -0.0000000000i	      0.6962791402 +       0.0000000000i	     -0.0077435146 +       0.0000000000i	     -0.0213636919 +       0.0000000000i	     -0.6839065257 +       0.0000000000i	      0.0226254784 +      -0.0000000000i	     -0.0997840458 +      -0.0000000000i	      0.7282559050 +      -0.0000000000i	      0.0105830566 +       0.0000000000i	      0.1037712077 +      -0.0000000000i	     -0.7427786956 +       0.0000000000i	      0.1477414698 +      -0.0000000000i	
     -0.0731741197 +       0.0000000000i	     -0.1550623924 +       0.0000000000i	      0.1531794104 +       0.0000000000i	      0.0030343837 +      -0.0000000000i	     -0.1121904730 +      -0.0000000000i	      0.1242051769 +      -0.0000000000i	      0.0310455069 +      -0.0000000000i	      0.1205887091 +      -0.0000000000i	     -0.1064056676 +       0.0000000000i	     -0.0521922876 +      -0.0000000000i	      0.1477414698 +       0.0000000000i	     -0.1709633849 +       0.0000000000i	

Phonon_frequencies (cm-1) for q = [      0.0000000000       0.0000000000       0.0000000000]:
      0.0000000000 +    1689.7924038545i	      0.0000000000 +     789.8664774637i	      0.0000000000 +     773.6947785354i	      0.0000000000 +     615.9810927041i	      0.0000000000 +     463.7699416195i	      0.0000000000 +     396.5319397300i	      0.0000000000 +     282.6398672150i	      0.0000000000 +     220.6451086109i	      0.0000000000 +     156.6021035994i	    140.1885806127 +       0.0000000000i	    331.5872179167 +       0.0000000000i	    594.7287735816 +       0.0000000000i	
 Phonon modes for q = [      0.0000000000       0.0000000000       0.0000000000]:
      0.0001917448	      0.5076174605	     -0.1540305049	     -0.0862319578	      0.2015366116	     -0.4591092645	     -0.0822203853	     -0.4856793704	     -0.3369738630	      0.2897912111	      0.1014619173	     -0.0964122079	
     -0.5028179829	     -0.0153501103	      0.0067006457	     -0.4895727148	      0.2061466629	     -0.0119461834	      0.2364816538	      0.2353516306	     -0.0423800830	      0.0092174787	     -0.1778692697	     -0.5654559908	
     -0.0299908613	     -0.0209758931	      0.4527485873	     -0.2556833312	      0.2946896161	     -0.0582785616	     -0.0900370416	     -0.1180440110	      0.6133716489	      0.4340263497	      0.0851345884	      0.2101769326	
     -0.0059391198	     -0.5098228101	      0.1467238474	     -0.1739546528	     -0.1286850383	      0.3932019720	      0.0785091261	     -0.5267219627	     -0.3790768155	      0.2829310703	      0.0772514459	     -0.0614245333	
      0.5067691128	     -0.0102903165	     -0.1353391105	     -0.2920335435	     -0.0373180337	      0.1381321684	     -0.6465370393	      0.1386590213	      0.0369784175	      0.1124981758	     -0.1313009442	     -0.3879511961	
      0.0083752707	     -0.0099237080	      0.4546614834	      0.2348148378	     -0.4959864842	     -0.3471901112	      0.0098258743	     -0.1786312068	      0.0980085651	      0.0114389623	     -0.4880720940	     -0.3024502984	
      0.1012541595	      0.4837188860	      0.1904128971	      0.0355746926	     -0.3307047963	      0.3571077869	      0.2529465252	      0.3608989771	     -0.1622826340	      0.4912437141	      0.1369272405	     -0.0267253658	
     -0.4973370025	      0.0933025697	      0.0474915178	      0.2754323180	      0.1835637145	      0.2578980988	     -0.5021374917	      0.0687663426	     -0.2119841100	      0.1767175176	     -0.4220234069	      0.2334182798	
     -0.0150852573	      0.0229075428	     -0.4155757027	      0.4863054722	      0.1458902829	      0.2737878460	      0.1727271975	     -0.2465019292	      0.4254624008	      0.2333934054	     -0.0613857309	     -0.4049085246	
     -0.0957282700	     -0.4744678812	     -0.2102744917	      0.2006099812	     -0.0866589814	     -0.4685399553	     -0.0737950047	      0.3750122591	     -0.1040233569	      0.5090000010	      0.1945390288	     -0.0197469504	
      0.4713808189	     -0.1137510183	      0.1146399730	      0.0976797705	      0.5283684726	     -0.0454435884	      0.3534938293	      0.1522702476	     -0.2437895724	      0.1166315273	     -0.4834385790	      0.0777532106	
     -0.0128993648	      0.0158770673	     -0.5060765054	     -0.3916881231	     -0.3392067428	     -0.0292972072	      0.1690470731	     -0.0816015130	      0.1766637064	      0.1871220892	     -0.4660672536	      0.3942712558	
