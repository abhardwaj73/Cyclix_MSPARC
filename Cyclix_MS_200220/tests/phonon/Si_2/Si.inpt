#CELL: 18.870000000000001 0.392699081698724 12.8999999999999
#TWIST_ANGLE: 0.030441789278971
#FD_GRID: 63 25 43
CELL: 20.000000000000000 0.392699081698724 8.001
FD_GRID: 80 19 32 #50 12 20 #67 16 40
KPOINT_GRID: 1 2 6
KPOINT_SHIFT: 0 0 0.5
BC: D C P
FD_ORDER: 12
EXCHANGE_CORRELATION: LDA_PZ
CHEB_DEGREE: 100
TOL_SCF: 1e-5
SMEARING: 0.001
ELEC_TEMP_TYPE: fd

QPOINT_GRID: 0 0 0