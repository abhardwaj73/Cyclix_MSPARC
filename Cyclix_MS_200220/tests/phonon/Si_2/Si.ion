#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name> <valence charge>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...


# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.

ATOM_TYPE: Si              # atom type followed with valence charge
N_TYPE_ATOM: 4               # number of atoms of this type
PSEUDO_POT: Si_local.psp8
COORD_FRAC:
0.588157665635968                   0                   0
0.588157665635968   0.000000000000000   0.333333333333333
0.588157665635968   0.500000000000000   0.500000000000000
0.588157665635968   0.500000000000000   0.833333333333333

#COORD_FRAC:
#1.005072284181751                   0                   0
#1.051177212639621   0.000000000000000   0.333333333333333
#1.005072284181751                 0.5                 0.5
#1.051177212639621   0.500000000000000   0.833333333333333



