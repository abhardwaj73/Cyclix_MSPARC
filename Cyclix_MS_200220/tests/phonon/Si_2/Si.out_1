***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 21-Mar-2022  Start time: 09:39:54                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 67 16 27
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 2 6
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_2/Si.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.298507 (Bohr)
Mesh spacing in y-direction        :  0.288711 (Bohr)
Mesh spacing in z direction        :  0.296333 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/phonon/Si_2/Si.out_1
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.84 0.61 7.11
Number of atoms of type 1          :  4
Estimated total memory usage       :  243.40 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.0322923670E+00        3.926E-01        33.564
2            -3.0937980651E+00        1.574E-01        8.448
3            -3.1076170770E+00        4.931E-02        7.944
4            -3.1083643550E+00        2.234E-02        8.083
5            -3.1086906823E+00        1.623E-02        7.833
6            -3.1087570923E+00        6.676E-03        7.926
7            -3.1087997177E+00        3.175E-03        7.802
8            -3.1088009863E+00        3.192E-03        7.843
9            -3.1088062350E+00        2.109E-03        7.732
10           -3.1088088925E+00        4.111E-04        8.828
11           -3.1088080464E+00        1.598E-04        7.439
12           -3.1088076652E+00        8.980E-05        7.299
13           -3.1088081828E+00        5.980E-05        7.242
14           -3.1088079257E+00        7.986E-05        7.283
15           -3.1088080540E+00        3.780E-05        7.179
16           -3.1088076703E+00        2.283E-05        7.568
17           -3.1088077233E+00        2.848E-05        10.542
18           -3.1088078598E+00        2.561E-05        8.579
19           -3.1088079394E+00        2.480E-05        10.867
20           -3.1088079815E+00        2.391E-05        8.766
21           -3.1088079349E+00        2.267E-05        7.565
22           -3.1088079062E+00        2.279E-05        7.206
23           -3.1088077705E+00        5.053E-05        7.190
24           -3.1088078675E+00        1.218E-05        7.358
25           -3.1088078496E+00        1.606E-05        7.113
26           -3.1088079828E+00        1.936E-05        7.046
27           -3.1088078903E+00        2.125E-05        7.179
28           -3.1088078932E+00        1.025E-05        7.226
29           -3.1088078658E+00        9.695E-06        7.099
Total number of SCF: 29    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.1088078658E+00 (Ha/atom)
Total free energy                  : -1.2435231463E+01 (Ha)
Band structure energy              : -6.9162479897E+00 (Ha)
Exchange correlation energy        : -6.1301655043E+00 (Ha)
Self and correction energy         : -5.1083195327E+01 (Ha)
Entropy*kb*T                       : -5.9650821578E-04 (Ha)
Fermi level                        : -1.4572341062E-01 (Ha)
Average force                      :  1.9005699527E-01 (Ha/Bohr)
Maximum force                      :  1.9006995964E-01 (Ha/Bohr)
Time for force calculation         :  5.141 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  277.463 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
