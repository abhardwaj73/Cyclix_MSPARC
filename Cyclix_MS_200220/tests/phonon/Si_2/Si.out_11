***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 23-Mar-2022  Start time: 15:51:04                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 67 16 27
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 6
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_2/Si.out_11
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.298507 (Bohr)
Mesh spacing in y-direction        :  0.288711 (Bohr)
Mesh spacing in z direction        :  0.296333 (Bohr)
Number of symmetry adapted k-points:  3
Output printed to                  :  ../tests/phonon/Si_2/Si.out_11
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.84 0.61 7.11
Number of atoms of type 1          :  4
Estimated total memory usage       :  162.71 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9730368634E+00        4.100E-01        27.325
2            -3.0556961173E+00        1.438E-01        6.739
3            -3.0680416305E+00        6.708E-02        6.620
4            -3.0721068600E+00        3.293E-02        6.397
5            -3.0721320791E+00        3.466E-02        6.362
6            -3.0719058987E+00        4.053E-02        6.497
7            -3.0721758721E+00        1.636E-02        6.610
8            -3.0721959971E+00        9.058E-03        6.424
9            -3.0721899942E+00        7.133E-03        6.378
10           -3.0722440855E+00        2.499E-03        6.484
11           -3.0722399620E+00        1.097E-03        6.090
12           -3.0722391325E+00        2.783E-04        6.005
13           -3.0722390631E+00        2.681E-05        5.983
14           -3.0722383277E+00        3.693E-05        5.915
15           -3.0722390739E+00        1.901E-05        5.919
16           -3.0722389051E+00        3.177E-06        5.857
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.0722389051E+00 (Ha/atom)
Total free energy                  : -1.2288955621E+01 (Ha)
Band structure energy              : -7.9228949602E+00 (Ha)
Exchange correlation energy        : -6.0820443591E+00 (Ha)
Self and correction energy         : -5.1083195327E+01 (Ha)
Entropy*kb*T                       : -4.1460879449E-04 (Ha)
Fermi level                        : -1.6955888036E-01 (Ha)
Average force                      :  1.8647078690E-01 (Ha/Bohr)
Maximum force                      :  1.8648632175E-01 (Ha/Bohr)
Time for force calculation         :  2.455 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  137.378 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
