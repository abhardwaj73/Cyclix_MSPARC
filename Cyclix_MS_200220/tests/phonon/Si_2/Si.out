***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 21-Mar-2022  Start time: 09:01:53                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 0.392699 8.001000 
FD_GRID: 67 16 27
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 6
KPOINT_SHIFT: 0 0 5.000000e-01
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 80
NSTATES: 15
EXCHANGE_CORRELATION: LDA_PZ
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/phonon/Si_2/Si.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.298507 (Bohr)
Mesh spacing in y-direction        :  0.288711 (Bohr)
Mesh spacing in z direction        :  0.296333 (Bohr)
Number of symmetry adapted k-points:  3
Output printed to                  :  ../tests/phonon/Si_2/Si.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  16
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.84 0.61 7.11
Number of atoms of type 1          :  4
Estimated total memory usage       :  162.71 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9730368634E+00        4.100E-01        35.497
2            -3.0556961173E+00        1.438E-01        7.937
3            -3.0680416305E+00        6.708E-02        8.074
4            -3.0721068600E+00        3.293E-02        7.981
5            -3.0721320793E+00        3.466E-02        7.591
6            -3.0719059004E+00        4.053E-02        7.215
7            -3.0721758721E+00        1.636E-02        7.568
8            -3.0721959970E+00        9.058E-03        8.496
9            -3.0721899943E+00        7.133E-03        7.641
10           -3.0722440856E+00        2.499E-03        7.014
11           -3.0722399620E+00        1.097E-03        7.351
12           -3.0722391325E+00        2.783E-04        7.359
13           -3.0722390631E+00        2.681E-05        8.104
14           -3.0722383261E+00        3.692E-05        7.635
15           -3.0722390733E+00        1.903E-05        7.248
16           -3.0722389047E+00        3.182E-06        7.178
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.0722389047E+00 (Ha/atom)
Total free energy                  : -1.2288955619E+01 (Ha)
Band structure energy              : -7.9228952299E+00 (Ha)
Exchange correlation energy        : -6.0820443611E+00 (Ha)
Self and correction energy         : -5.1083195327E+01 (Ha)
Entropy*kb*T                       : -4.1460883131E-04 (Ha)
Fermi level                        : -1.6955889737E-01 (Ha)
Average force                      :  1.8647078734E-01 (Ha/Bohr)
Maximum force                      :  1.8648632166E-01 (Ha/Bohr)
Time for force calculation         :  3.109 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  179.628 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
